README.md

```diff
@@ -1,6 +1,102 @@
 # SMART Health Links Designs
 
-This work provides SMART Health Links (https://docs.smarthealthit.org/smart-health-links/) and the implementers with (1) guidance on appropriate display and use of SMART Health Links and (2) reference designs.
+This work provides SMART Health Links (https://docs.smarthealthit.org/smart-health-links/) and implementers with (1) guidance on appropriate display and use of SMART Health Links and (2) reference designs. 
 
-The overall objective is to encourage consistent visual representations of SMART Health Links to increase recognition among users.
+The overall objective is to encourage consistent visual representations of SMART Health Links to increase recognition among users. Use of reference designs is not a requirement, but we certainly encourage it where appropriate.
 
+Note that SMART Health Links are under active development. These guidelines and designs will change as needed to reflect updates to the spec as they are made. 
+
+## Designs
+
+The following designs are available as PNG and PSD files. More formats will follow. 
+
+| File | Description |
+|-----------|---------|
+| [SMART Health Link Setup](Generic_Mobile_UX_Link_Set-up.png) | Setup screen for a particular SMART Health Link, allowing the user to view or configure expiration and passcode and to activate the link  |
+| [SMART Health Link Management](Generic_Mobile_UX_Link_Manage.png) | Sharing and management screen for a particular SMART Health Link, allowing the user to view, copy, and share the link as well as view and manage access |
+| [SMART Health Link QR](Generic_Mobile_UX_SHL_QR.png) | QR code display screen for a particular SMART Health Link |
+| [SMART Health Link Access Log](Generic_Mobile_UX_Access_Log.png) | Access log screen for a particular SMART Health Link |
+
+## Key UX Elements
+
+### SMART Health Link Setup
+Provide users with a clear overview of the SMART Health Link they plan to create (depicted in [SMART Health Link Setup](Generic_Mobile_UX_Link_Set-up.png)). Include,
+
+* Label for the link (e.g. “Jane’s Complete Immunizations”)
```

Comment:
Should we clarify that this is the same as the label embedded in the SHL itself? In other words, the value here needs to be a <80 char string that will be visible to anyone who obtains the SHL, **even before they demonstrate knowledge of the passcode**? Or are we imagining this as a private label in the management UX distinct from what, if anything, is embedded in the SHL?

Reply:
i wonder if a little vocab section would be useful?